src/test/kotlin/com/exactpro/th2/codec/TestDecodeProcessor.kt

```diff
@@ -98,13 +101,27 @@ internal class TestDecodeProcessor {
         val messageID = MessageID.newBuilder()
             .setSequence(1)
             .build()
-        assertThrows<DecodeException> {
-            processor.process(RawMessage.newBuilder().setBody(ByteString.copyFrom(rawData)).apply { metadataBuilder.id = messageID }.build())
-        }
+
+        val builder = processor.process(RawMessage.newBuilder().setBody(ByteString.copyFrom(rawData)).apply { metadataBuilder.id = messageID }.build())[0]
+        assertEquals("th2-codec-error", builder.metadata.messageType)
```

Comment:
Use constant instead of string 
```suggestion
        assertEquals(ERROR_TYPE_MESSAGE, builder.metadata.messageType)
```